src/components/Footer/index.jsx

```diff
@@ -7,15 +7,35 @@ import {
   faInstagram,
 } from "@fortawesome/free-brands-svg-icons";
 import "./index.css";
-import { Link } from "react-router-dom";
+import { Link, useLocation } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 import globe from "./../../images/earthNav.svg";
 
+const LIGHT_GREEN_BACKGROUND_PAGES = [
+  "/about",
+  "/resources",
+  "/games/memorygame",
+  "/games/puzzlegame",
+  "/games/quizgame",
+];
+
+const LIGHT_GREEN = "#b4d0c4";
+
```

Comment:
delete this newline. these constants are logically grouped together

Reply:
PTAL